### PR TITLE
feat: Added sound effect to win and draw state.

### PR DESCRIPTION
**Pull request by shivam-sharma-03**
**Description**:
This Pull Request is a significant enhancement for the malivinayak Tic Tac Toe project, for the issue #19 , introducing an exciting new feature: Sound effect when the player win the game or when the game is a draw. Leveraging my expertise in HTML, CSS, and JavaScript, I've meticulously added these sound effects for enjoyable user experience also i have modified the look/ interface of the game to give it a modern look .

**Changes Made**:

Game Interface: Changed the UI of the game by giving the boxes different color adding borders shadows etc. . This will provide modern and engaging gameplay experience.

Sound Effects: To elevate the overall gaming experience, I've integrated sound effects that trigger when the game concludes. This addition adds an immersive dimension to the game, enhancing the player's enjoyment.

**Related Issue**:
This Pull Request directly addresses and resolves issue #19 .

**Checklist**:
Before merging this Pull Request, I have diligently ensured the following:

- [x] The code adheres to the project's coding style and conventions.
- [x] The Tic Tac Toe game's styling has been meticulously reviewed for cross-browser compatibility.

**Screenshots**:
I have included screenshots of the Tic Tac Toe game in action to provide a visual representation of the changes made. You can access these screenshots in the "Screenshots" section of this Pull Request.

**Testing**:

- [x] I have thoroughly tested the Tic Tac Toe game to verify that it functions flawlessly.
- [x] The sound effects have been tested and confirmed to play appropriately upon game conclusion.
- [x] The game's compatibility across various browsers has been validated through testing.

**Additional Context**:
Please review the changes introduced in this Pull Request. The addition of the Tic Tac Toe game significantly enhances the project's interactivity and appeal. Your feedback is highly appreciated, and once approved, this exciting feature will be integrated into the project.

Checklist:

- [x] I have reviewed the code changes and am confident they are correct.
- [x] I have tested the Tic Tac Toe game thoroughly to ensure it functions as intended.
- [x] I have provided relevant context and details in this PR description.
- [x] I have attached screenshots to visually showcase the changes.
- [x] I have confirmed the project's coding style and cross-browser compatibility.
- [x] I have linked this PR to the related issue #19 .

**Screenshots**:

Tic Tac Toe
Tic Tac Toe1

### CODE REVIEW

File: winsound.mp3
Comment (malivinayak) [inline winsound.mp3:1]: Add this file at assets/sound

File: gametie.mp3
Comment (malivinayak) [inline gametie.mp3:1]: Add this file at assets/sound

File: assets/js/script.js
Comment (malivinayak) [inline assets/js/script.js:1]: Looks Good!👍

**Review by malivinayak — CHANGES_REQUESTED** [review]
You have done really good job.
Due to multiple contributions currently this branch has merge conflicts.
<details><summary>Solution:</summary>
<p>
You can rebase the branch</p>
</details> 

Once branch conflicts are resolved and requested minor changes are done request the review again so that I can merge the PR after review

**Comment by shivam-sharma-03** [comment]
Thank you for your feedback doing the described things right away.
Also can you merge this pr as soon as possible because the hactoberfest themselves have a seven day review period, which will take a lot of time and ten it'll let me know whether this PR will count or not.
________________________________

File: gametie.mp3
Comment (shivam-sharma-03) [reply gametie.mp3:1]: Added the sounds to the assets ->sound.

**Comment by shivam-sharma-03** [comment]
hey @malivinayak  I have completed the requested task.
please review this PR again.
issue #19 .

**Review by malivinayak — CHANGES_REQUESTED** [review]
https://github.com/malivinayak/Tic-Tac-Toe/assets/66154908/d77914c5-3019-4db3-87b4-ca2af0278b5f

I have encountered following bugs, you have to resolve this:
- [ ] Player's turn is not updating
- [ ] Rules button is not working anymore
- [ ] After win/draw, reload perform but after that player turn label doesn't show up unless we manually refresh the page

**Comment by shivam-sharma-03** [comment]
resolved the conflicts @malivinayak

**Comment by shivam-sharma-03** [comment]
I have fixed following bugs:

[x]Player's turn is not updating
[x]Rules button is not working anymore
[x]After win/draw, reload perform but after that player turn label doesn't show up unless we manually refresh the page

**Pull request closed without merging**